src/components/tag/Tag.vue

```diff
@@ -31,6 +31,7 @@
             role="button"
             :aria-label="ariaCloseLabel"
             class="delete is-small"
+            :class="[{'tag is-paddingless is-rounded' : closeType}, closeType]"
```

Comment:
Why?

Reply:
For changing color, it should be a `.tag` element first like attached one. Otherwise Bulma's own .delete class' color would be active only https://github.com/jgthms/bulma/blob/master/css/bulma.css#L55
And `is-paddingless` + `is-rounded` combination needed for making tag rounded.

Reply:
It looks like a workaround.. i think it's better add a custom implementation in the tags.scss, for example:
```
.tag {
    .delete  {
        @each $name, $pair in $colors {
            $color: nth($pair, 1);
            &.is-#{$name}:before {
              background: $color;
            }
        }
    }
}
```

Reply:
I aggree. However :before and :afters for cross' design by default in Bulma. Though we can still write custom css for `is-delete` and `delete` classes. See my commit and let me know your opinons. https://github.com/buefy/buefy/pull/2492/commits/f0673d9afcba61a17242cb45c9cda859e6d64319 @jtommy